docs/getting_started/install_manually.md

```diff
@@ -198,6 +198,10 @@ a2dissite 000-default.conf
 service apache2 restart
 ```
 
+!!! note "Restart server"
+    
+    This commend may vary depending on your Linux distribution.
```

Comment:
Typo in `command`. Also, I'd repeat restarting server inside the note's body, because people tend to ignore titles, something like: ` At this point you need to restart the server. This command...`